test/unit/storage_async_test.rb

```diff
@@ -251,7 +251,7 @@ def test_tls_no_client_certificate
     config_obj = {
       url: 'rediss://localhost:46379',
       ssl_params: {
-        ca_file: File.expand_path(File.join(__FILE__, '..', '..', '..', 'script', 'config', 'ca-root-cert.pem'))
+        ca_file: Tempfile.new('ca-root-cert.pem').tap { _1.write(create_cert.to_pem) && (_1.flush ; _1.close) }.to_path
```

Comment:
apparently earlier OpenSSL silently ignored missing CA files. Newer OpenSSL fails. I mean the C library.

Reply:
Added a new commit, should be good now.